src/annotator/components/test/ProfileModal-test.js

```diff
@@ -0,0 +1,75 @@
+import { act } from 'preact/test-utils';
```

Comment:
This file is mostly copied from `NotebookModal-test.js`, minus the things not relevant for the profile.